include/Gaffer/Private/IECorePreview/TaskMutex.h

```diff
@@ -187,24 +187,7 @@ class TaskMutex : boost::noncopyable
 
 					m_mutex->m_executionState->arena.execute(
 						[this, &fWrapper] {
-#if TBB_INTERFACE_VERSION >= 10003
```

Comment:
So Maya 2019.2 (latest release) is using TBB_INTERFACE_VERSION 9106 (tbb 2017.0)... I'm inclined to go with the hack, since we won't be onto a new-enough Maya for years it seems...

I might still take the opportunity to sort out the standalone Gaffer builds at IE though.

Reply:
>  I'm inclined to go with the hack, since we won't be onto a new-enough Maya for years it seems...

:man_facepalming: :cry:

> I might still take the opportunity to sort out the standalone Gaffer builds at IE though.

:tada: That would be great - I'm sure we're not done with the CachePolicy/TBB stuff, and more recent versions of TBB give us more options there. While you're considering this, what do you think it would take to start using the official builds for standalone Gaffer at IE?